src/Http/BaseApplication.php

```diff
@@ -82,7 +82,7 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
         $cakeResponse = $this->getDispatcher()->dispatch($cakeRequest, $cakeResponse);
 
         // Convert the response back into a PSR7 object.
-        return $next($request, ResponseTransformer::toPsr($cakeResponse));
+        return ResponseTransformer::toPsr($cakeResponse);
```

Comment:
How would they do that? The app is bound after all the middleware processing is done.